sys/arm64/arm64/pmap.c

```diff
@@ -845,6 +857,9 @@ pmap_bootstrap_dmap(vm_offset_t kern_l1, vm_paddr_t min_pa,
 				KASSERT(l2_slot != 0, ("..."));
 				pmap_store(&l2[l2_slot],
 				    (pa & ~L2_OFFSET) | ATTR_DEFAULT |
+#if __has_feature(capabilities)
+				    ATTR_CAP_RW |
```

Comment:
Would it make sense to define this as 0 for `!__has_feature(capabilities)`?

Reply:
We haven't done that elsewhere, and technically it can still be defined as non-zero for a non-CHERI kernel so long as you know your CPU still supports CHERI (but you've just decided to not compile in support), but you'd want the uses to be guarded.

Reply:
For MD pmap bits in RISC-V and MIPS we have only used the bits under #define's and always defined them to their real values.  For the MI VM_PROT bits we always define non-zero values for VM_PROT_READ/WRITE_CAP and rely on non-CHERI architectures to just ignore the values.